ansible/roles/prosody/defaults/main.yml

```diff
@@ -139,6 +139,7 @@ prosody_meet_auth_vpaas_enabled: false
 prosody_meet_av_moderation_enabled: true
 prosody_meet_chat_history_url: false
 prosody_meet_conf_duration_enabled: true
+prosody_get_media_type: true
```

Comment:
There are environments where this is not needed and not used, so let's keep it false by default and enable it only where needed.